src/forms/validators/default-value-validator.ts

```diff
@@ -0,0 +1,25 @@
+import moment from 'moment';
+import { FieldValidator, OHRIFormField } from '../types';
+
+export const OHRIDefaultFieldValueValidator: FieldValidator = {
+  validate: (field: OHRIFormField, value: any) => {
+    if (['radio', 'checkbox', 'select', 'content-switcher'].includes(field.questionOptions.rendering)) {
```

Comment:
Please add the list to a constant

Reply:
Well I'm happy to do this:

```
 const codedTypes = ['radio', 'checkbox', 'select', 'content-switcher'];
 if (codedTypes.includes(field.questionOptions.rendering)) {
    // ..
 }

```
but I think it's redundant variable since it's used once, but it's more readable I guess, let me do that.